core/src/main/java/org/axonframework/messaging/StreamableMessageSource.java

```diff
@@ -35,4 +38,47 @@
      */
     MessageStream<M> openStream(TrackingToken trackingToken);
 
+    /**
+     * Creates the token at the beginning of an event stream.
```

Comment:
Perhaps it's good to be explicit about the fact that "beginning" is where the oldest events live?